package.json

```diff
@@ -213,7 +213,7 @@
     "@babel/preset-react": "7.10.4",
     "@babel/preset-typescript": "7.10.4",
     "@babel/traverse": "^7.15.0",
-    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/jest-dom": "^5.16.1",
```

Comment:
Just to double-check: did we discuss that it is okay to upgrade this here?

Reply:
This was not discussed because this dependency was added to test React Table, is not present in development, the upgrade is fine.